### PR TITLE
Speedmeter Small cleanup

### PR DESCRIPTION
**Pull request by hensldm**
Noticed a couple things will doing MM's version.
The rename to `x` is more just my interpretation of what it probably was, though don't really have proof. If preferred I can revert it back to unk.

### CODE REVIEW

**Review by fig02 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Comment by fig02** [comment]
has met merge requirements but will merge tomorrow incase anyone else wants to take a look

**Review by Dragorn421 — APPROVED** [review]

File: src/code/speed_meter.c [hunk 1]
```diff
@@ -237,10 +248,9 @@ void SpeedMeter_DrawAllocEntries(SpeedMeter* meter, GraphicsContext* gfxCtx, Gam
         y++;
     }
 
-    thga = (TwoHeadGfxArena*)&state->tha;
-    //! @bug THA_GetRemaining call should be THGA_GetRemaining like the others below, harmless as-is
```
Comment (Thar0) [inline src/code/speed_meter.c:241]: Why is the bug comment gone?

**Review by Thar0 — CHANGES_REQUESTED** [review]

File: src/code/speed_meter.c (hunk 1 above)
Comment (fig02) [reply src/code/speed_meter.c:241]: he changed the temp to be a tha instead of a thga, so its the right function call

**Review by Thar0 — APPROVED** [review]

File: src/code/speed_meter.c (hunk 1 above)
Comment (Dragorn421) [reply src/code/speed_meter.c:241]: I was very confused as well, I think we missed the tha to thga cast back when you (tharo) overhauled th(g)a docs

**Pull request merged**